pkg/apis/extensions/types.go

```diff
@@ -114,11 +133,7 @@ type HorizontalPodAutoscalerStatus struct {
 	// CurrentConsumption is the current average consumption of the given resource that the autoscaler will
 	// try to maintain by adjusting the desired number of pods.
 	// Two types of resources are supported: "cpu" and "memory".
-	CurrentConsumption *ResourceConsumption `json:"currentConsumption"`
-
-	// LastScaleTimestamp is the last time the HorizontalPodAutoscaler scaled the number of pods.
-	// This is used by the autoscaler to controll how often the number of pods is changed.
-	LastScaleTimestamp *unversioned.Time `json:"lastScaleTimestamp,omitempty"`
+	CurrentMetricUtilizations []MetricUtilization `json:"currentMetricUtilizations"`
```

Comment:
Would also benefit from the patch tags if you think there's any chance you'd want to be able to update this via patch rather than replacing the whole object.